sdcm/cluster_gce.py

```diff
@@ -338,16 +321,7 @@ def node_setup(self, node, verbose=False):
             # detect pd-ssd and pd-standard disks
             result = node.remoter.run('ls /dev/sd[b-z]')
             disks_str = ",".join(re.findall('/dev/sd\w+', result.stdout))
```

Comment:
Can we wrap the code to detect disks in a function? it will be easy to maintain.
The code is repeated inside node.scylla_setup()